docs/src/snap/howto/epa.md

```diff
@@ -0,0 +1,1061 @@
+# How to set up Enhanced Platform Awareness
```

Comment:
Pros - easier to follow chunks
Cons - some parts are inextricably linked together, e.g. hugepages and rtk which makes splitting it into logical sections harder
     - the testing section would also need to be split
     - it would likely need a lot more work and there are other things to do

I would vote for fixing this page and publishing it first, we can make a card to disentagle it and create a ctaegory for it when it is actually in a usable state.